schema/codelists/partyRole.csv

```diff
@@ -4,11 +4,11 @@ procuringEntity,Procuring entity,"The organization managing the contracting (or
 supplier,Supplier,An organization with which a buyer or a procuring entity decided to conclude a contract.,,
 tenderer,Tenderer,An organization that submitted a bid.,,
 funder,Funder,The funder is an organization providing money or finance for this contracting process.,,1.2
-enquirer,Enquirer,An organization that has made an enquiry during the enquiry phase of a contracting process.,,
+enquirer,Enquirer,An organization that submitted an enquiry during the enquiry period of the contracting process.,,
 payer,Payer,An organization making a payment from a transaction.,,
 payee,Payee,An organization in receipt of a payment from a transaction.,,
 reviewBody,Review body,An organization responsible for the review of this procurement process. This organization often has a role in any challenges made to the contract award.,,
-interestedParty,Interested party,"An organization that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",,
+interestedParty,Interested party,"An organization that has expressed an interest in the contracting process: for example, by signing up for notifications about the process, submitting questions or downloading or purchasing procurement documents.",,
```

Comment:
The examples suggest that the party is a bidder, but in #557 we clarified that it can be anything – so adding a sentence for further clarity here.

Using "enquiries" consistently to avoid longer "questions or requests for clarification".

Using "requesting" instead of two words.

```suggestion
interestedParty,Interested party,"An organization that expressed interest in the contracting process: for example, by signing up for notifications, submitting enquiries, or requesting procurement documents. The organization can be a potential supplier, news organization, investor, etc.",,
```